runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -45,6 +45,8 @@ public Layout getCurrentLayout() {
         while (true) {
             try {
                 return runtime.layout.get();
+            } catch (SystemUnavailableException sue) {
```

Comment:
This shouldn't be necessary, the error will not be caught by the exception catch block below